### DISCUSSION TITLE
Gigabyte GeForce RTX™ 4080 16GB AERO OC Fan speed issue

### DISCUSSION THREAD

**Discussion by NazZaR** [Rem0o/FanControl.Releases#2236] [Q&A]
Hello!
I have some weird issue (or not) with my Gigabyte GeForce RTX™ 4080 16GB AERO OC. 
Usually for RTX cards in idle, fan stop is  at around 30% speed. But with this Gigabyte 4080, it is around 52% for some reason. And it does some weird stop-speed up constantly if set below 52%. It comes to almost full stop and then speeds up rapidly to over 1500RPM and then calms down. And this cycle continues unless you set it to minimum 52% speed, which is a lot for GPU without load.
I don't know, whether it is by design or I am doing something wrong.

Here is a video of GPU behavior when set under 52% speed.
https://github.com/Rem0o/FanControl.Releases/assets/5753530/314bef1f-cd5f-4857-bb22-c082df772420

**Comment by NazZaR** [comment]
Turns out it is probably some of strange Gigabyte decisions. This fan speed behavior is by design.
https://www.reddit.com/r/gigabyte/comments/zukxv6/gigabyte_4080_have_some_hardcoded_minimum_fan/

**Reply by Rem0o** [reply]
What kind of start/ stop % did you set on the control card for the GPU?

**Reply by Rem0o** [reply]
From your reddit thread it seems like there is some built in behaviour you got to play around with as you mentioned.

**Comment by NazZaR** [comment]
> What kind of start/ stop % did you set on the control card for the GPU?

I left the values that Fan Control detected:
![image](https://github.com/Rem0o/FanControl.Releases/assets/5753530/2e3eb0a9-5b77-4aa8-b7ef-c6e84df8f65b)
